### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by PeterChav01**
# Description
Created the sidebar for navigation between sections of store and employee eachone with their own views and forms 
  
  ## Screenshots 
image
image

### CODE REVIEW

File: clock/app/helpers/main_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module MainHelper
+end
```
Comment (OswaldoPineda) [inline clock/app/helpers/main_helper.rb:2]: is this file necessary? if not, pls delete it

File: clock/app/javascript/packs/application.js [hunk 2]
```diff
@@ -26,4 +26,29 @@ $(document).ready(function() {
   $(".close").click(function() {
     $("#Errors").toggle("slow");
   });
-});
\ No newline at end of file
+});
+<<<<<<< HEAD
+=======
```
Comment (OswaldoPineda) [inline clock/app/javascript/packs/application.js:31]: you have some merge conflicts here, pls solve them first
this could be helpful
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

File: clock/app/javascript/packs/application.js [hunk 3]
```diff
@@ -26,4 +26,29 @@ $(document).ready(function() {
...
+
+let getDOM = (id) => {
+  return document.getElementById(id)
+ }
+
+ window.onload = function() {
+  const tabs = document.querySelectorAll("#navTabs>li>a"),
+    panes = document.querySelectorAll("#navPanes>div")
+
+  for (let i = 0; i < tabs.length; i++)
```
Comment (OswaldoPineda) [inline clock/app/javascript/packs/application.js:41]: use the method `forEach` instead of for

File: clock/app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -4,12 +4,18 @@
...
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
+  <div class="row">
+    <%= render 'shared/sidebar' %>
+    <div class="col-9">
+      <%= yield %>
+		</div>
```
Comment (OswaldoPineda) [inline clock/app/views/layouts/application.html.erb:18]: check the indentation here

File: clock/app/views/shared/_sidebar.html.erb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        </li>
+        <div class="sb-sidenav-menu-heading">Stores</div>
+        <li class="nav-item">
+          <%= link_to 'Show', stores_path, class: "nav-link" %>
+				</li>
+        <li class="nav-item">
+          <%= link_to 'Create', new_store_path , class: 'nav-link'%>
+				</li>
+			</ul>
+		</div>
```
Comment (OswaldoPineda) [inline clock/app/views/shared/_sidebar.html.erb:25]: check the indentation on this file, and add an extra line at the end of the file

**Review by OswaldoPineda — CHANGES_REQUESTED** [review]

**Review by Ilyeo — APPROVED** [review]
looks good to me :shipit:

**Review by OswaldoPineda — APPROVED** [review]
